packages/react-data-grid/src/ReactDataGrid.js

```diff
@@ -427,7 +430,7 @@ const ReactDataGrid = React.createClass({
     }
 
     if (this.props.onGridRowsUpdated) {
-      this.onGridRowsUpdated(cellKey, toRow, toRow, {[cellKey]: textToCopy}, AppConstants.UpdateActions.COPY_PASTE);
+      this.onGridRowsUpdated(cellKey, toRow, toRow, {[cellKey]: textToCopy}, AppConstants.UpdateActions.COPY_PASTE, fromRow);
```

Comment:
should this not be 

```
this.onGridRowsUpdated(cellKey, fromRow, toRow, {[cellKey]: textToCopy}, AppConstants.UpdateActions.COPY_PASTE);
```

Does that not make the logic of this.onGridRowsUpdated a lot easier as well?